package.json

```diff
@@ -36,7 +36,6 @@
     "leaflet-draw": "github:enketo/Leaflet.draw#ff730785db7fcccbf2485ffcf4dffe1238a7c617"
   },
   "overrides": {
-    "__note_1__": "See https://github.com/supabase/supabase-js/issues/936",
```

Comment:
nit(blocking): I'm not sure why this got removed but it would be good to keep. It's not actually valid code for the `package.json`, but because you are not able to leave comments this is how I've left a note-to-self to remind why the version of supabase auth is defined the way it is in the block below